giving powers to the bot

neo-fetch opened the pull request.

neo-fetch committed:
Added kick functionality to v_2

imported some stuff too

neo-fetch committed:
Update MaidSama.py

IceCereal reviewed (changes requested):
Three things:
Is this discord.py rewrite?
Secondly, could you add a reason as a parameter of the function kick?
`++kick UserID "Trolling"` should be the way the command should work.
Can you change the name from `kick` to `kickOut`

neo-fetch committed:
Did the 3rd thing

And I need you to join my server back to test my bot to give a msg after being kicked out

IceCereal reviewed (changes requested):
I can't merge it as long as it's not rewrite.

IceCereal commented:
I'm closing this PR until this changes to rewrite.
